docs/mkdocs/docs/faq.md

```diff
@@ -40,24 +40,55 @@ of both batch and streaming data.
 - Support for "dynamic schemas" - ArcticDB supports datasets with changing schemas (column sets) over time.
 - Support for automatic data deduplication.
 
-###*What sort of data is ArcticDB best suited to?*
+### *What sort of data is ArcticDB best suited to?*
 
 ArcticDB is an OLA(nalytical)P DBMS, rather than an OLT(ransactional)P DBMS.
 
 In practice, this means that ArcticDB is optimised for large numerical datasets
 and for queries that operate over many rows at a time.
 
-###*Does ArcticDB require a server?*
+### *Does ArcticDB require a server?*
 
 No. ArcticDB is a fully fledged embedded analytical database system,
 designed for modern cloud and on-premises object storage that does not require
 a server for any of the core features.
 
-###*What languages can I use ArcticDB with?*
+### *What languages can I use ArcticDB with?*
 
 Bindings are currently only available for Python. 
 
-###*How can I get started using ArcticDB?*
+### *What is the best practice for saving Data to ArcticDB?*
+
+Users are encouraged to store data in its original vendor format initially, then optimize storage based on their specific use cases, which may include maintaining multiple versions of the data to support different analysis needs. In ArcticDB, best practices for data storage involve considering the balance between read and update performance, data layout’s impact on index performance, and auditability against original data sources. It’s recommended to store data in a format that facilitates efficient querying. 
+
+A full guide can be found [here](tutorials/data_organisation.md)!
+
+### *What are the Limitations of ArcticDB being client-side?*
+
+The serverless nature of ArcticDB provides excellent performance, making it ideal for data science applications where speed and efficiency are key. Although it may not be ideal for scenarios that demand strong transactionality and concurrent updates, ArcticDB excels in complex, multi-tenant environments where its attributes align closely with those of popular NoSQL databases such as MongoDB and Cassandra, offering flexibility and scalability. 
```

Comment:
I would probably delete the "and concurrent updates" clause. We are good at concurrent updates on different data sets for example.